pkgs/development/compilers/ponyc/default.nix

```
@@ -94,12 +94,13 @@ stdenv.mkDerivation (rec {
 
   installPhase = "make config=release prefix=$out "
     + lib.optionalString stdenv.hostPlatform.isDarwin ("bits=64 " + (lib.optionalString (!lto) "lto=no "))
```

This looks super confusing:

```suggestion
  installPhase = "
```